core/controllers/suggestion.py

```diff
@@ -103,13 +109,54 @@ class SuggestionHandler(base.BaseHandler):
 
     @acl_decorators.can_suggest_changes
     def post(self):
+        suggestion_type = self.payload.get('suggestion_type')
+        change_dict = self.payload.get('change')
+        filenames = []
+        if (
+                suggestion_type ==
+                suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT):
+            html_list = [change_dict['translation_html']]
```

Comment:
Controller layer should not be aware of the internals of this dict. Can we do somthing for this? [Maybe use the suggestion class get_all_html_content method?]

Reply:
Not sure! [I'll think about this a bit and share what we can do here!]